src/System.Device.Gpio.Tests/GpioControllerTestBase.cs

```diff
@@ -7,30 +7,16 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
-[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
Why did you remove this? I believe we still need our tests to not run in parallel.

Reply:
Nvm I just saw you moved it.